packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -4,6 +4,10 @@
 
 // @dart=2.9
 
+import 'dart:convert';
+import 'dart:html' as html;
```

Comment:
we cannot import dart:html directly as that will break our flutter desktop build. The good news is, it looks like you are just using this to download the file, which we already have infrastructure to do. See import_export.dart and look at class ExportController. Here is an example usage of this in the cpu profiler screen: https://github.com/flutter/devtools/blob/7885be593b17d6f0c93c5874395c5e500b39fecf/packages/devtools_app/lib/src/screens/profiler/profiler_screen_controller.dart#L88